common/script/content.coffee

```diff
@@ -55,6 +55,24 @@ api.mystery =
   wondercon: {start:'2014-03-24',end:'2014-04-01'} # not really, but the mechanic works
 _.each api.mystery, (v,k)->v.key = k
 
+equipmentList =
```

Comment:
A more accurate name would be: `gemPurchasableItems`

You'll also need to put it on the `api` object so it'll be exported correctly: `api.gemPurchasableItems`